src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathMapperTest.java

```diff
@@ -805,7 +805,7 @@ public void testQueryElementFollowingSibling() throws Exception {
                 .findFirstToken(TokenTypes.METHOD_DEF)
                 .getNextSibling();
         final DetailAST[] expected = {expectedMethodDefNode,
-                expectedMethodDefNode.getNextSibling()};
+            expectedMethodDefNode.getNextSibling()};
```

Comment:
please restore indentation.

Reply:
Here, I encounter problem, whenever I generate snapshot of changes by `mvn --batch-mode -Pno-validations clean package -Passembly` and check on this file no errors are encountered. This error is encourtered only during `mvn clean verify` at `ant-phase-verification`.

Reply:
This problem is linked with https://github.com/checkstyle/checkstyle/pull/8038/files#r415130563
removing those changes doesn't require this change, those changes doesn't have effect on internal testcases it only affects `mvn clean verify (ant-phase verify)`.

Reply:
> please restore indentation.

Done.